t/lib/Dashboard/Test.pm

```diff
@@ -31,14 +31,257 @@ sub new ($class, %options) {
   return $self;
 }
 
+sub cleanup_fixtures ($self, $app) {
```

Comment:
There is quite a huge amount of code to add jobs. Is all the data necessary in each job or can for the purpose of tests some data be removed?

Reply:
Honestly, i would actually prefer to test with an even larger data set, closer to production. Right now it's all still very synthetic.

Reply:
Sure, I don't mind to use a big dataset for testing. I meant just the amount of code for creating the fixture jobs. Maybe you can use a for-loop to create a list of jobs.

Reply:
But i can reuse the other (minimal) dataset too for less code. Long term i'd like to have at least a few hundred entries per table serialised in the repo to bootstrap test scenarios.